build_steps.py

```diff
@@ -459,6 +505,21 @@ def generate_installer(platform):
         )
     ]
 
+@inject_paths
```

Comment:
I'm not so sure about this. Nimsuggest has [it's own repository](https://github.com/nim-lang/nimsuggest), and in any case, it would probably be better to split testing of tools and such into another build that is run on successful completion of the main Nim build.